website/docs/docs/dbt-versions/release-notes.md

```diff
@@ -20,8 +20,24 @@ Release notes are grouped by month for both multi-tenant and virtual private clo
 
 ## June 2024
 
-- **New:** [Job warnings](/docs/deploy/job-notifications) are now GA. Previously, you could receive email or Slack alerts about your jobs when they succeeded, failed, or were canceled. Now with the new **Warns** option, you can also receive alerts when jobs have encountered warnings from tests or source freshness checks during their run. This gives you more flexibility on _when_ to be notified. 
+#### dbt Semantic Layer
+- **New:** Introduced a new granularity option for cumulative metrics, `period_agg`, nested under the `type_params` parameter. It specifies how to roll up the cumulative metric to another granularity. Options are `start`, `end`, `average`. Defaults to `start` if no `window` is specified.
+- **New:** Added validation that derived metrics must always have an `expr` key.
+- **New:** Enabled cache to pickup where filters that are included in saved queries.
+- **Enhancement:** [Cumulative metrics](/docs/build/cumulative) now support the `cumulative_type_params` key. The `cumulative_type_params` parameter replaces `type_params`. MetricFlow supports the old `type_params` key but will issue a warning if used.
```

Comment:
I think we can just strike this. It's not an enhancement in and of it's self, but an additional parameter needed to support cumulative granularity.

Reply:
ok got it @Jstein77 , do i need to add this to the docs though? we only have `type_params` in the doc: https://docs.getdbt.com/docs/build/cumulative